lib/messager.rb

```diff
@@ -0,0 +1,21 @@
+class Messager
+
+  require 'twilio-ruby'
+  require 'dotenv'
+
+  def initialize
+    @credentials = Dotenv.load
+  end
+
+  def send_sms(message)
+
+    @client = Twilio::REST::Client.new @credentials["ACCOUNT_ID"], @credentials["AUTH_TOKEN"]
+
+    @client.account.messages.create({
+    	:from => @credentials["PHONE_FROM"],
+    	:to => @credentials["PHONE_TO"],
```

Comment:
Style/HashSyntax: Use the new Ruby 1.9 hash syntax. (https://github.com/bbatsov/ruby-style-guide#hash-literals)